internal/builders/maven/action.yml

```
@@ -54,7 +54,7 @@ runs:
   steps:
     - uses: actions/checkout@96f53100ba2a5449eb71d2e6604bbcd94b9449b5 # v 3.5.2
     - name: Set up JDK
-      uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2 # v 3.11.0
+      uses: actions/setup-java@b943a4ed876ec91f6d0cde2181d88d24e8e348de # v 3.11.0
```

We don't want this to update to HEAD
```suggestion
      uses: actions/setup-java@cd89f46ac9d01407894225f350157564c9c7cee2 # v3.12.0
```